functions/command/add.go

```diff
@@ -26,15 +30,38 @@ func NewAddCommand(guildID, activityType, name string) *AddCommand {
 
 func (c *AddCommand) Execute(ctx context.Context, cl *clients.Clients) (*discordgo.WebhookEdit, error) {
 	var typ activity.ActivityType
+	var info *activity.GameInfo
 	switch c.ActivityType {
 	case "activity":
 		typ = activity.ACTIVITY
 	case "game":
-		return utils.NewWebhookEdit("🚧 Games not implemented yet"), nil
+		typ = activity.GAME
+
+		ctx, cancel := context.WithTimeout(context.Background(), time.Duration(time.Millisecond*500))
+		defer cancel()
+
+		detail, err := cl.Rawg().GetGame(ctx, c.Name)
+
+		if err != nil {
+			if rawgError, ok := err.(*rawg.RawgError); ok && rawgError.HttpCode == http.StatusNotFound {
+				return utils.NewWebhookEdit("🚧 Game not found 🚧"), err
+			} else {
+				return utils.NewWebhookEdit("🤖🔥 Bot error 🔥"), err
```

Comment:
```suggestion
				return nil, err
```
We have a default error in the Command function handler code